common/app/conf/switches.scala

```diff
@@ -789,6 +788,15 @@ object Switches {
     exposeClientSide = false
   )
 
+  val IdentityCookieRefreshSwitch = Switch(
```

Comment:
Yes, I'd rather keep this around for now (just in-case we see unforeseen issues).